Pollination-apps/Pollination-apps

```diff
@@ -0,0 +1,26 @@
+# Pollination Apps
+
+A Pollination app is a way to capture a workflow in a web application.
+Examples of these workflows are customized weather data visualization,
+running simulations on models prepared in a CAD environment, post-processing
+the results of a simulation, and visualizing the results back into the CAD environment.
+[These](https://pollination.gallery/) are some of the early examples of the apps that
+give an idea about what's possible with the apps.
+
```

Comment:
We need a paragraph here to discuss that Pollination allows you to deploy your apps and share them with your team easily so you don't have to worry about the infrastructure and deployment.